stages/stage_2/compile.py

```diff
@@ -120,7 +120,7 @@ def do_com_prog(prog, macro_lookup, symbol_table, run_program):
         # (com ((OP) . RIGHT)) => (a (com (q OP)) 1)
         inner_exp = eval(prog.to([b"com",
             quote(operator), quote(macro_lookup), quote(symbol_table)]), TOP.as_path())
-        return prog.to(inner_exp)
+        return prog.to([inner_exp])
```

Comment:
Oh man, my eye lingered on these two lines in the previous change, but I didn't see this. Sorry about that.

How come this worked before? Lemme look at the history

Reply:
Ah, I see. Looks like perhaps this was conflated with the removal of `((`

Reply:
There's some code in this area I no longer completely understand. I feel like I should do a complete rewrite of `stage2` as a warm-up for `stage3`, which I want to be 100% clvm (besides parsing to IR).